File: MyMindNotes/app/src/main/java/com/android/mymindnotes/model/retrofit/CheckNicknameApi.java
package com.android.mymindnotes.model.retrofit;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface CheckNicknameApi {
    @GET("/api/member/checkNickname/{nickname}")
    Call<Map<String, Object>> checkNickname(@Path("nickname") String nickname);
}

File: MyMindNotes/app/src/main/java/com/android/mymindnotes/data/retrofit/CheckNickNameApi.kt
package com.android.mymindnotes.data.retrofit

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface CheckNickNameApi {
    @GET("/api/member/checkNickname/{nickname}")
    suspend fun checkNickname(
        @Path("nickname") nickname: String
    ): Map<String, Object>
}




